agent/agent_test.go

```diff
@@ -970,6 +970,96 @@ func TestAgent_SCP(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestAgent_FileTransferBlocked(t *testing.T) {
+	t.Parallel()
+
+	assertFileTransferBlocked := func(t *testing.T, errorMessage string) {
+		// NOTE: Checking content of the error message is flaky. It can catch different responses.
+		isErr := strings.Contains(errorMessage, agentssh.BlockedFileTransferErrorMessage) ||
+			strings.Contains(errorMessage, "EOF") ||
+			strings.Contains(errorMessage, "Process exited with status 2")
```

Comment:
for reviewers:

I know it is nasty, but we can't control which error is returned. We know that transfer will be chopped for sure, and one of the symptoms will appear.

Reply:
I don't understand this; could you add some more context please?
I think expanding the code comment (rather than this thread) would be useful for future maintenance.

Reply:
SSH client may end up with following error messages:
"EOF"
"unexpected EOF"
"Process exited with status 2"
<there was one more stating that Init Packet failed due to ...blah blah EOF>
and of course <BlockedFileTransferErrorMessage>

I suspect that sometimes the error message might be shadowed with EOF, but I didn't analyze it. It is failing this way on CI.